docs/installation.rst

```diff
@@ -119,7 +100,7 @@ Ingestion tasks populate the database with version control push logs, queued/run
 
   .. code-block:: bash
 
-     (venv)vagrant@local:~/treeherder$ celery -A treeherder worker -B
+     (venv)vagrant@local:~/treeherder$ celery -A treeherder worker -B --concurrency 5
```

Comment:
I wonder if we can get away with `-c 5` and say above in the docs, *", start up 5 concurrent worker threads to process async tasks"*.. or similar. Just in case new contributors think there's only one verbose argument format.